packages/nx-sonarqube/src/executors/scan/utils/utils.ts

```diff
@@ -118,6 +118,8 @@ export async function scanner(
       'sonar.scm.provider': 'git',
       'sonar.sources': paths.sources,
       'sonar.sourceEncoding': 'UTF-8',
+      'sonar.tests': paths.sources,
```

Comment:
@jwmickey I decided to just pass in sources for now. We can address it if the use case comes up where tests live in a different folder

Reply:
Sounds good, thank you!